auth-web/public/config/configuration.json

```diff
@@ -1,16 +1,16 @@
 {
   "AUTH_URL": "https://dev.bcregistry.ca/business/auth/",
-  "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_LEGAL_ROOT_API": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_PATH_NEW_BUSINESS": "https://business-create-dev.pathfinder.gov.bc.ca/businesses/",
+  "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "VUE_APP_LEGAL_ROOT_API": "https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "VUE_APP_STATUS_ROOT_API": "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1",
+  "VUE_APP_PATH_NEW_BUSINESS": "https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/",
   "DIRECTOR_SEARCH_URL": "https://dev.bcregistry.ca/directorsearch/",
-  "VUE_APP_AUTH_RESET_API": "http://auth-api-dev.pathfinder.gov.bc.ca/test/reset",
+  "VUE_APP_AUTH_RESET_API": "https://auth-api-dev.apps.silver.devops.gov.bc.ca/test/reset",
   "LAUNCH_DARKLY_ENV_KEY": "5db9da115f58e008123cd783",
   "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86",
   "BCEID_OSD_LINK": "https://www.test.bceid.ca/os/?7686&SkipTo=Basic",
-  "FILE_SERVER_URL":"http://minio-1rdehl-dev.pathfinder.gov.bc.ca/public",
+  "FILE_SERVER_URL":"https://minio-dev.apps.silver.devops.gov.bc.ca/public",
```

Comment:
I copied the new values from the Dev config. Is "https" correct for local use? Is the URL correct for local use?